src/app/tests/Extensions.ts

```diff
@@ -81,40 +84,52 @@ expect.extend({
         };
     },
 
-    // toBeConnectedTo(source: unknown, target: DigitalComponent, options = { depth: Infinity }) {
-    //     if (!(source instanceof DigitalComponent))
-    //         throw new Error("toBeConnectedTo can only be used with DigitalComponents!");
-
-    //     const { depth } = options;
-
-    //     const visited = new Set<DigitalComponent>();
-    //     function bfs(layer: DigitalComponent[], depth: number): boolean {
-    //         if (depth === 0 || layer.length === 0)
-    //             return false;
-
-    //         const queue = [] as DigitalComponent[];
-    //         for (const cur of layer) {
-    //             visited.add(cur);
-
-    //             const connections = [
-    //                 ...cur.getOutputs().map((w) => w.getOutputComponent()),
-    //                 ...cur.getInputs().map((w) => w.getInputComponent()),
-    //             ].filter((c) => !visited.has(c));
-
-    //             if (connections.includes(target))
-    //                 return true;
-
-    //             queue.push(...connections);
-    //         }
-    //         return bfs(queue, depth-1);
-    //     }
-
-    //     const pass = bfs([source], depth);
+    toBeOk(received: unknown) {
```

Comment:
You can just say `received: Result` and then you don't have to type-check it, I should probably do that for the others but I was probably referencing some js